### ISSUE TITLE
Write access on the filesystem not working

### ISSUE THREAD

**Issue by ArneDBKey2info** [versionpress/support#104]
- VersionPress version: 4.0-beta
- WordPress version: 4.8
- OS, webserver, PHP: XAMPP
- Git version: 1.9
- Operation System: Mac - Mac osX

For som reason I keep getting Write access on the filesystem error when trying to activate VersionPress:
VersionPress needs write access in the site root, its nested directories and the system temp directory. Please update the permissions.

I've tried putting everything from the root and below on 777. But The error keeps displaying.

**Comment by ArneDBKey2info** [comment]
More information regarding the permission:

```
array (
  'exec-user' => 'daemon',
  'proc_open-user' => 'daemon',
  'php-can-write' => 
  array (
    'ABSPATH' => true,
    'WP_CONTENT_DIR' => true,
    'sys_temp_dir' => false,
  ),
  'php-can-delete' => 
  array (
    'ABSPATH' => true,
    'WP_CONTENT_DIR' => true,
    'sys_temp_dir' => true,
  ),
  'process-can-write' => 
  array (
    'ABSPATH' => true,
    'WP_CONTENT_DIR' => true,
    'sys_temp_dir' => false,
  ),
  'php-can-delete-file-created-by-process' => 
  array (
    'ABSPATH' => true,
    'WP_CONTENT_DIR' => true,
    'sys_temp_dir' => true,
  ),
)
```

**Comment by borekb** [comment]
The `daemon` user doesn't have write pemissions to the system temp directory, other than that, everything seems OK.

**Comment by ArneDBKey2info** [comment]
How do I give him write permission in the system temp directory? I'm kinda noob at this matter.

**Comment by borekb** [comment]
`chmod`, Server Fault and sites like that are a great place to get help with this.

**Comment by pavelevap** [comment]
Closed as duplicate of #106
